src/render/opengles2/SDL_render_gles2.c

```diff
@@ -714,7 +723,7 @@ GLES2_QueueDrawLines(SDL_Renderer * renderer, SDL_RenderCommand *cmd, const SDL_
     prevy = 0.5f + points->y;
     *(verts++) = prevx;
     *(verts++) = prevy;
-    *((int *)verts++) = color;
+    *((SDL_Color*)verts++) = color;
```

Comment:
Maybe verts should point to a type safe structure definition?

Reply:
Yes that would probably be good in the future. For this PR I just did the minimal change needed to synchronize the backend code with the higher level code.